README.md

```diff
@@ -165,4 +165,4 @@ Before submitting a pull request, please make sure the following is done:
 3. Ensure the test suite passes (`npm test`).
 
 ###Style Guide
```

Comment:
This isn't part of your PR, but I see that we need a space between the ### and Style, if we put it together it will render the ### characters and not the Style Guide as a header.